cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -61,53 +80,68 @@ func CreateKubeDNSRBACClusterRole(clientset *clientset.Clientset) error {
 	if _, err := clientset.Rbac().ClusterRoles().Create(&clusterRole); err != nil {
 		return err
 	}
-
-	subject := rbac.Subject{
-		Kind:      "ServiceAccount",
-		Name:      master.KubeDNS,
-		Namespace: metav1.NamespaceSystem,
-	}
-
-	clusterRoleBinding := rbac.ClusterRoleBinding{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "kubeadm:" + master.KubeDNS,
-		},
-		RoleRef: rbac.RoleRef{
-			APIGroup: "rbac.authorization.k8s.io",
-			Kind:     "ClusterRole",
-			Name:     clusterRole.Name,
-		},
-		Subjects: []rbac.Subject{subject},
-	}
-	if _, err := clientset.Rbac().ClusterRoleBindings().Create(&clusterRoleBinding); err != nil {
-		return err
-	}
-	fmt.Println("[apiconfig] Created kube-dns RBAC rules")
-
 	return nil
 }
 
-// CreateKubeProxyClusterRoleBinding grants the system:node-proxier role to the nodes group,
-// since kubelet credentials are used to run the kube-proxy
-// TODO: give the kube-proxy its own credential and stop requiring this
-func CreateKubeProxyClusterRoleBinding(clientset *clientset.Clientset) error {
-	clusterRoleBinding := rbac.ClusterRoleBinding{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "kubeadm:node-proxier",
+// CreateClusterRoleBindings creates all
+func CreateClusterRoleBindings(clientset *clientset.Clientset) error {
+	clusterRoleBindings := []rbac.ClusterRoleBinding{
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "kubeadm:kubelet-bootstrap",
+			},
+			RoleRef: rbac.RoleRef{
+				APIGroup: "rbac.authorization.k8s.io",
```

Comment:
Well, you've been creating them all along so why not follow the same pattern everywhere?